develop/tutorials/code/dynamic-data-mapping-type-time/src/main/resources/content/Language.properties

```diff
@@ -1,4 +1,6 @@
 mask=Mask
 placeholder-text=Placeholder Text
 time-description=time Description
-time-label=time Label
\ No newline at end of file
+time-label=time Label
```

Comment:
I think this line is no long necessary, right?

Reply:
Right!